scripts/start/calculate_CES_configuration.R

```diff
@@ -5,7 +5,7 @@
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
 calculate_CES_configuration <- function(cfg, path = getwd(), check = FALSE) {
-    CESstring <- paste0("indu_", cfg$gms$industry,ifelse(cfg$gms$cm_subsec_model_steel=="processes","pbs",""),"-",
+    CESstring <- paste0("indu_", cfg$gms$industry,ifelse(cfg$gms$cm_subsec_model_steel=="ces","CesSteel",""),"-",  # remove CesSteel suffix after process-based has been default for some months
```

Comment:
You are redefining the meaning of the `CESstring`.  This is going to bite some project team when they have the least time to deal with it …

Reply:
the change of teh meaning is intended as pds will be our default from now on